### DISCUSSION TITLE
Migrating from Acr.Settings

### DISCUSSION THREAD

**Discussion by dyeazel** [shinyorg/shiny#1430] [Q&A]
I have an older project that uses Acr.Settings. That repo says the library moved to Shiny, but I find no information here on how to migrate. Can anyone point me to some doc's or a guide?

**Comment by aritchie** [comment]
I don't offer a migration from libraries because this isn't a product.  You likely won't want to take in a library like Shiny just for settings anyhow.  

Shiny.Core has Stores which line up with the ISettings.  IKeyValueStore for secure storage, local settings, and in-memory.  I don't document these classes because they're mostly meant to be used internally.  For the best way of using settings with Shiny, you can use https://shinylib.net/client/other/statefulservices/
